File: src/python/readme_reader.py
import os

def read_readme(filepath="../../README.md"):
    """Reads the README.md file and returns its content."""
    if not os.path.exists(filepath):
        raise FileNotFoundError("README.md not found.")
    
    with open(filepath, "r", encoding="utf-8") as file:
        return file.read()

if __name__ == "__main__":
    print(read_readme())

File: src/typescript/readme_reader.ts
import { promises as fs, existsSync } from 'fs';
import { join } from 'path';

export async function readReadme(filePath: string = "../../README.md"): Promise<string> {
    if (!existsSync(filePath)) {
        throw new Error("README.md not found.");
    }
    return fs.readFile(filePath, 'utf-8');
}

if (require.main === module) {
    readReadme()
        .then(content => console.log(content))
        .catch(error => console.error(error.message));
}
